docs/Projects/TAD/ai_assesment_tool_checklist.md

```diff
@@ -0,0 +1,155 @@
+# ALTAI
```

Comment:
can you nest these files a little deeper (in a "tools" directory or something)? 

And maybe we should shift all of this to a "comparison" directory anyway?